View and add resources by admin

armant opened the pull request:
r @weronica @dzhang55

weronica commented on app/single_resource/forms.py:
```diff
@@ -0,0 +1,14 @@
+
+
+class CreateResourceForm(Form):
+    name = StringField('Name', validators=[InputRequired(), Length(1, 64)])
```
Can we maintain the same style for listing validators?
```
validators=[
   ...
]
```

weronica commented:
@armant can you post screenshots of the UIs that you created?

dzhang55 commented:
lgtm

armant commented:
r @dzhang55 @weronica 

I rebased from master. Seems like Git treated all the commits at master that happened after the point when this branch originally branched off from master (i.e. all the commits that got applied to this branch during the rebase) as commits of this branch too, and thus shows them here as well. For reference, here's the list of files that I actually changed:

	modified:   app/assets.py
	new file:   app/assets/scripts/vendor/address-autocomplete.js
	modified:   app/single_resource/forms.py
	modified:   app/single_resource/views.py
	modified:   app/templates/single_resource/create.html
	new file:   app/templates/single_resource/edit.html
	modified:   app/templates/single_resource/index.html
	new file:   app/templates/single_resource/resource_form.html

armant commented:
![screen shot 2016-04-01 at 1 10 55 am](https://cloud.githubusercontent.com/assets/6414384/14198314/ceccc604-f7a6-11e5-88ee-669efde06fa1.png)
![screen shot 2016-04-01 at 1 11 04 am](https://cloud.githubusercontent.com/assets/6414384/14198316/d49203b0-f7a6-11e5-8630-98a7fb998f68.png)
![screen shot 2016-04-01 at 1 11 17 am](https://cloud.githubusercontent.com/assets/6414384/14198319/d9030c5a-f7a6-11e5-905c-532c11ea89a0.png)
![screen shot 2016-04-01 at 1 11 27 am](https://cloud.githubusercontent.com/assets/6414384/14198322/dcb0cc52-f7a6-11e5-816d-87f504d53949.png)

weronica commented on app/templates/single_resource/index.html:
```diff
@@ -1,7 +1,29 @@
-  </div>
+    <div class="ui text container">
+        <h2 class="ui dividing header">Resources</h2>
+        <a href="/single-resource/create">Add</a>
```
Use `url_for` function here

armant commented:
Some of the comments from meeting:
-truncate long/lat when showing in a table
-have explicit table headers in front end instead of `field_name`

weronica commented:
@armant Instead of "have explicit table headers in front end instead of `field_name`", I meant "have explicit table headers in back-end and then pass them in a list to the front-end."

weronica commented on app/single_resource/views.py:
```diff
@@ -1,8 +1,71 @@
+                           resources=resources)
+
+
+@single_resource.route('/create', methods=('GET', 'POST'))
```
Should be `['GET', 'POST']` instead of `('GET', 'POST')`

weronica commented on app/single_resource/views.py:
```diff
@@ -1,8 +1,71 @@
+    return render_template('single_resource/create.html', form=form)
+
+
+@single_resource.route('/<int:resource_id>', methods=('GET', 'POST'))
```
Should be `['GET', 'POST']` instead of `('GET', 'POST')`

weronica commented on app/templates/single_resource/resource_form.html:
```diff
@@ -0,0 +1,13 @@
+} %}
+
+
+{{ f.render_form(form) }}
```
Newline please :)

armant replied:
It's there, doesn't show for some reason

dzhang55 commented:
looks good
